test/PasskeyAccount.t.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: UNLICENSED
+// Force a specific Solidity version for reproducibility.
+pragma solidity ^0.8.13;
+
+import { Test } from "forge-std/Test.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { console2 } from "forge-std/console2.sol";
+
+import { IEntryPoint } from "@aa/interfaces/IEntryPoint.sol";
+import {
+    UserOperation, UserOperationLib
+} from "@aa/interfaces/UserOperation.sol";
+
+import { PasskeyAccount, Base64Url } from "src/account/PasskeyAccount.sol";
+import { PasskeyAccountFactory } from "src/account/PasskeyAccountFactory.sol";
+
+contract PasskeyAccountTest is Test {
+    using UserOperationLib for UserOperation;
+
+    // Get the entryPoint instance from the local node
+    IEntryPoint public entryPoint =
+        IEntryPoint(0x5FbDB2315678afecb367f032d93F642f64180aa3);
+    // Get the entryPoint instance from the mainnet or Sepolia testnet.
+    // IEntryPoint public entryPoint =
+    //     IEntryPoint(0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789);
+
+    string constant testCredId = "9h5F3DgLSjSMdnVOadmhCw";
+    uint256 constant testPubKeyX =
+        67299174900712686363169673082376821529726602378544032702281553676098545184711;
+    uint256 constant testPubKeyY =
+        104273800132786176334597151467609377740095818152192999025225464410568038480397;
+    uint256 constant testSalt = 0;
```

Comment:
nit: The `test` prefix has no much meaning. It would be better to remove it

```suggestion
    string constant credId = "9h5F3DgLSjSMdnVOadmhCw";
    uint256 constant pubKeyX =
        67299174900712686363169673082376821529726602378544032702281553676098545184711;
    uint256 constant pubKeyY =
        104273800132786176334597151467609377740095818152192999025225464410568038480397;
    uint256 constant salt = 0;
```